gradle/src/main/groovy/com/flurry/android/symbols/SymbolUploadPlugin.groovy

```diff
@@ -119,4 +116,20 @@ class SymbolUploadPlugin implements Plugin<Project> {
 
         return configValues
     }
+
+    /**
+     * Returns the mapping file
+     * @param variant the ApplicationVariant
+     * @return the file or null if not found
```

Comment:
is this `@return` documentation correct? (not sure what the behavior is if we don't have a mapping file provider)